source/install/install-kubernetes-mattermost.rst

```diff
@@ -107,3 +105,123 @@ Use your domain registration service to create a canonical name or IP address re
+      host: ""
+      username: ""
+      password: ""
+
+The Mattermost installation manifest contains fields which must be edited in line with your configuration and environment requirements.
+
+.. csv-table::
+    :header: "Field", "Description", "Must Edit"
+
+    "metadata.name", "The name of your Mattermost as it will be shown in Kubernetes. The shorter the better.", "Yes"
+    "spec.size", "The size of your installation. This can be '100users', '1000users, '5000users', '10000users', or '25000users' Custom values are accepted provided the replication formula is correct.", "Yes"
+    "spec.ingressName", "The DNS for your Mattermost installation. This is the same name as the existing deployment, but the underlying resource is the restored database.", "Yes"
+    "spec.version", "The Mattermost version.", "No"
+    "spec.mattermostLicenseSecret", "The name of the Kubernetes secret containing your license (e.g. mattermost-license). Required for Enterprise deployments.", "Yes"
+    "spec.database.storageSize", "The storage size for your database. Your Kubernetes cluster must have volumes this size or larger.", "No"
+    "spec.minio.storageSize", "The storage size for your file storage. Your Kubernetes cluster must have volumes this size or larger.", "No"
+    "spec.elasticSearch", "The section for Elasticsearch settings. Remove this section if you will not be using Elasticsearch.", "No"
+    "spec.elasticSearch.host", "The hostname for your Elasticsearch instance. Remove this section if you will not be using Elasticsearch.", "No"
+    "spec.elasticSearch.username", "The username for your Elasticsearch instance. Remove this section if you will not be using Elasticsearch.", "No"
+    "spec.elasticSearch.password", "The password for your Elasticsearch instance. Remove this section if you will not be using Elasticsearch.", "No"
+
+7. Create a restore manifest:
+
+Open a text editor and create a text file with the following details. Save the file as ``restore.yaml``:
+
+.. code-block:: yaml
+
+  apiVersion: mattermost.com/v1alpha1
+  kind: MattermostRestoreDB
+  metadata:
+    name: example-mattermostrestoredb
+  spec:
+    initBucketURL: s3://my-sample/my-backup.gz
+    mattermostClusterName: example-clusterinstallation
+    mattermostDBName: mattermostdb
+    mattermostDBPassword: supersecure
+    mattermostDBUser: ""
+    restoreSecret: ""
+
+**Parameters**
+- ``mattermostClusterName``. The ClusterInstallation file name.
+- ``restoreSecret``. The location of the backup file.
+- ``mattermostDBPassword``. The password used to access the database.
+- ``mattermostDBUser``. The username required to access the database.
+- ``initBucketURL``. The URL of the storage instance/server where the backed up DB is stored.
+
+8. To initiate deployment, apply the file and specify the path where the newly-created files have been saved:
+
+.. code-block:: sh
+
+      $ kubectl create ns mattermost
+      $ kubectl apply -n mattermost -f /path/to/restore.yaml
```

Comment:
This only covers the commands for the restore manifest. I think we need the same thing for the Mattermost cluster installation and the restore secret.

```
kubectl create ns mattermost
kubectl apply -n mattermost -f /path/to/secret.yaml
kubectl apply -n mattermost -f /path/to/mattermost-installation.yaml
kubectl apply -n mattermost -f /path/to/restore.yaml
```

Reply:
Thanks, @gabrieljackson! Apologies for the delayed reply. I've added this to the restore section.